lxd/container_lxc.go

```diff
@@ -1408,9 +1410,10 @@ func (c *containerLXC) deviceStaticShiftMounts(mounts []deviceConfig.MountEntryI
 
 // deviceAddCgroupRules live adds cgroup rules to a container.
 func (c *containerLXC) deviceAddCgroupRules(cgroups []deviceConfig.RunConfigItem) error {
+	cg, _ := c.cgroup(nil)
```

Comment:
We probably should still check `err` just to be safe, we don't want to change code in `cgroup` and suddenly be calling this function on `nil`.